edge/pkg/edged/apis/types.go

```diff
@@ -18,9 +21,39 @@ const (
 	StatusTag = "StatusTag"
 )
 
+// ContainerState is container state
+type ContainerState int32
+
+// Constants to show container status
+const (
+	ContainerCreated ContainerState = 0
+	ContainerRunning ContainerState = 1
+	ContainerExited  ContainerState = 2
+	ContainerUnknow  ContainerState = 3
+	ContainerPaused  ContainerState = 4
+)
+
+// Constants to check status of container state
+const (
+	StatusUnknown                              = kubecontainer.ContainerStateUnknown
+	StatusCreated                              = kubecontainer.ContainerStateCreated
+	StatusRunning                              = kubecontainer.ContainerStateRunning
+	StatusExited                               = kubecontainer.ContainerStateExited
+	StatusStopped kubecontainer.ContainerState = "stopped"
+	StatusPaused  kubecontainer.ContainerState = "paused"
```

Comment:
kubecontainer have no these two state?

Reply:
Why do we need these two status by the way?

Reply:
I have removed these constants, replace with kubecontainer state